selgym/gym.py

```diff
@@ -175,6 +175,17 @@ def linux_default_firefox_profile_path() -> str:
             return ospath.join(profile_path, entry)
     return None
 
+def mac_default_firefox_profile_path() -> str:
+    """Retrieves .default-release profile path on MacOS"""
+    profile_path = ospath.expanduser("~/Library/Application Support/Firefox/Profiles")
+
+    if not ospath.exists(profile_path):
+        raise RuntimeError(f"\nUnable to retrieve {profile_path} directory")
+
+    for entry in listdir(profile_path):
+        if ".default-release" in entry:
```

Comment:
Better to have more consistency across other functions using `.endswith()` instead of `in` keyword